### PR TITLE
Pale Beings no longer scream or jitter when hit by electrode and a custom message is displayed.

### PR DESCRIPTION
**Pull request by DeityLink**
Also replaced a few manifested ghost type checks with proper species macro checks.

:cl:
* tweak: Pale Beings no longer scream or jitter when hit by electrode since they don't feel pain, and a custom message is displayed to highlight the elctrode's ineffectiveness.

### CODE REVIEW

File: code/modules/projectiles/projectile/change.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 /obj/item/projectile/change/proc/wabbajack(var/mob/living/M,var/type) //WHY: as mob in living_mob_list
 	if(istype(M, /mob/living) && M.stat != DEAD)
-		if(istype(M, /mob/living/carbon/human/manifested)) // DEEEEEEEEEEEEEEITY
+		if(ismanifested(M))
```
Comment (jknpj) [inline code/modules/projectiles/projectile/change.dm:19]: DEEEEEEEEEEEEEEITY

File: code/datums/gamemode/factions/legacy_cult/soulstone.dm [hunk 2]
```diff
@@ -24,8 +24,8 @@
 /obj/item/device/soulstone/attack(var/mob/living/M, mob/user as mob)
 	if(!istype(M, /mob/living/carbon) && !istype(M, /mob/living/simple_animal))
 		return ..()
-	if(istype(M, /mob/living/carbon/human/manifested))
-		to_chat(user, "The soul stone shard seems unable to pull the soul out of that poor manifested ghost back onto our plane.")
+	if(ismanifested(M))
+		to_chat(user, "\The [src] shard seems unable to pull the soul out of that powerful body.")
```
Comment (jknpj) [inline code/datums/gamemode/factions/legacy_cult/soulstone.dm]: shouldn't it just be \the [src]

Comment (DeityLink) [reply code/datums/gamemode/factions/legacy_cult/soulstone.dm]: ` An object expression will display the object's name preceded by the text macro \the or \The if no other article has been specified.`

File: code/modules/projectiles/projectile/change.dm (hunk 1 above)
Comment (DeityLink) [reply code/modules/projectiles/projectile/change.dm:19]: who even wrote that and why?

Comment (DeityLink) [reply code/modules/projectiles/projectile/change.dm:19]: Granted I wonder if it's still a needed check considering that manifested ghosts aren't simple to produce en masse anymore.

File: code/modules/projectiles/projectile/energy.dm [hunk 3]
```diff
@@ -21,6 +21,9 @@
 	hitsound = 'sound/weapons/taserhit.ogg'
 
 /obj/item/projectile/energy/electrode/hit_apply(var/mob/living/X, var/blocked)
+	if (ismanifested(X))
+		X.visible_message("<span class='danger'>\The [X] seems to completely ignore \the [src] that hit him.</span>","<span class='warning'>You can barely feel at all \the [src]'s electrical discharge.</span>")
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline code/modules/projectiles/projectile/energy.dm]: \Them, not him

Comment (DeityLink) [reply code/modules/projectiles/projectile/energy.dm]: fix'd

File: code/datums/gamemode/factions/legacy_cult/soulstone.dm (hunk 2 above)
Comment (jknpj) [reply code/datums/gamemode/factions/legacy_cult/soulstone.dm]: no, i mean, drop the shard after [src] unless you want a soul stone shard shard

Comment (jknpj) [reply code/datums/gamemode/factions/legacy_cult/soulstone.dm]: They might be ranting about the not-really-oop way of handling this

Comment (DeityLink) [reply code/datums/gamemode/factions/legacy_cult/soulstone.dm]: OH

**Pull request merged**